### DISCUSSION TITLE
Multiple relationships to same table

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#3854] [Help]
Hello,

I'm trying to add multiple fields relationships to the same table (MySQL).
More details : 
- I have a table with countries
- I have a table with users
- My user can have **birth country**, **nationality country** and **origin country**
- I create a column with **birth country** in users table (one country => many users)
- Works well
- I create a column with **nationality country** in users table (one country => many users)
- The data isn't displayed anymore : No rows found (console message | 400 "Not unique table/alias: 'countries'")

Obviously I can create multiple table for each country selection but this doesn't make sense as the countries are exactly the same.

How can I achieve this?
Thanks for your help !

**Comment by diogenesbrussels** [comment]
Is there a follow-up to this or a way around this limitation?
Thank you!

**Reply by mjashanks** [reply]
Sorry for missing this.

We have not added any features around this - though it's a very valid use case.

Your only workaround is to use custom queries to `SELECT` / `UPDATE` the values in a separate query. Even when saving the data, your button actions could look like
1. `Save Row`
2. `Execute Query` - which runs `UPDATE person SET nationalityCountry = {{ country }} WHERE id = {{ id }}`

So - you could still make it happen in the "Save` button click. A bit of a pain though.

This is a totally valid use case of course, would be good to support this - but I can't see us doing it soon.

**Reply by diogenesbrussels** [reply]
Thank you for your feedback!
I'll keep an eye on this discussion for further improvement. I hope to be able to use the power of relationships for this one day!
